src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a specific field of contact information of a person. 
+ * Subclasses : Address, Phone, Email. 
```

Comment:
Good job for adding the header comment, but you don't need to say where this class is being used.